pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "fortress"
+version = "0.1.0"
+description = "for fetching secrets from Hashicorp Vault"
+authors = ["Artsy Product Sapphire Team <sapphire@artsymail.com>"]
+
+[tool.poetry.dependencies]
+boto3 = "^1.26.61"
+python = ">=3.9,<3.11"
```

Comment:
I think some of zenith ci is still running on 3.8.6, so this will be a motivation to update that

Reply:
i can try to lower min requirement to 3.8 and see if there are build failures. hokusai had a lot of build failures on 3.8, and 3.9 even.

Reply:
I think it's a good time to upgrade on the zenith side

Reply:
then i'll leave it as it is here.